locales/tr/tr.json

```diff
@@ -1,618 +1,618 @@
-    "Only Trashed": "Only Trashed",
-    "Original": "Original",
+    "Only Trashed": "Sadece Çöpe",
+    "Original": "Orijinal",
     "Page Expired": "Sayfa zaman aşımına uğradı",
     "Pagination Navigation": "Sayfalandırma Çubuğu",
     "Pakistan": "Pakistan",
-    "Palau": "Palau",
-    "Palestinian Territory, Occupied": "Palestinian Territories",
+    "Palau": "Palauname",
+    "Palestinian Territory, Occupied": "Filistin Toprakları",
     "Panama": "Panama",
-    "Papua New Guinea": "Papua New Guinea",
+    "Papua New Guinea": "Papua Yeni Gine",
     "Paraguay": "Paraguay",
     "Password": "Parola",
-    "Pay :amount": "Pay :amount",
-    "Payment Cancelled": "Payment Cancelled",
-    "Payment Confirmation": "Payment Confirmation",
-    "Payment Successful": "Payment Successful",
-    "Pending Team Invitations": "Pending Team Invitations",
-    "Per Page": "Per Page",
+    "Pay :amount": "Ödeme :amount",
+    "Payment Cancelled": "Ödeme İptal Edildi",
+    "Payment Confirmation": "Ödeme Onayı",
+    "Payment Successful": "Ödeme Başarılı",
+    "Pending Team Invitations": "Bekleyen Takım Davetiyeleri",
+    "Per Page": "Sayfa Başına",
     "Permanently delete this team.": "Bu ekibi temelli sil",
     "Permanently delete your account.": "Hesabını temelli sil",
     "Permissions": "İzinler",
     "Peru": "Peru",
-    "Philippines": "Philippines",
+    "Philippines": "Filipinler",
     "Photo": "Resim",
-    "Pitcairn": "Pitcairn Islands",
+    "Pitcairn": "Pitcairn Adaları",
     "Please click the button below to verify your email address.": "E-posta adresinizi doğrulamak için lütfen aşağıdaki düğmeye tıklayın.",
     "Please confirm access to your account by entering one of your emergency recovery codes.": "Girişinizi onaylamak için lütfen hesap kurtarma kodlarınızdan birini girin.",
     "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Lütfen kimlik doğrulayıcı uygulamanız tarafından sağlanan kimlik doğrulama kodunu girerek hesabınıza erişimi onaylayın.",
     "Please confirm your password before continuing.": "Devam etmeden önce lütfen parolanızı onaylayın.",
     "Please copy your new API token. For your security, it won't be shown again.": "Lütfen yeni API jetonunuzu kopyalayın. Güvenliğiniz için bir daha gösterilmeyecek.",
-    "Please enter your password to confirm you would like to log out of your other browser sessions across all of your devices.": "Please enter your password to confirm you would like to log out of your other browser sessions across all of your devices.",
+    "Please enter your password to confirm you would like to log out of your other browser sessions across all of your devices.": "Tüm cihazlarınızda diğer tarayıcı oturumlarınızdan çıkmak istediğinizi onaylamak için lütfen şifrenizi girin.",
```

Comment:
`"Please enter your password to confirm you would like to log out of your other browser sessions across all of your devices.": "Tüm cihazlarınızdaki diğer tarayıcı oturumlarınızı sonlandırmak istediğinizi onaylamak için lütfen şifrenizi girin.",`